infrastructure/smart-contracts/contracts/upgrades/referee/Referee10.sol

```diff
@@ -1106,6 +1112,21 @@ contract Referee10 is Initializable, AccessControlEnumerableUpgradeable {
 		}
     }
 
+    function updateBulkSubmissionOnTransfer(address from, address to, uint256 keyId) external {
```

Comment:
Why do we need to pass the `keyId`, its not used anywhere.

Reply:
It has been removed.